rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -278,6 +278,16 @@ http {
 
     {{ buildOpentracing $cfg $servers }}
 
+    {{ if $cfg.EnableOpentracing }}
```

Comment:
Please move this block inside [buildOpentracing](https://github.com/kubernetes/ingress-nginx/blob/ed68e4aeb50f7a074169a1cae9644291b941cac8/internal/ingress/controller/template/template.go#L941) so we can test this without e2e tests too

Reply:
done